framework/src/modules/pos/schemas.ts

```diff
@@ -731,3 +731,14 @@ export const getPendingUnlocksResponseSchema = {
 		},
 	},
 };
+// getRegistrationFeeResponseSchema
```

Comment:
This comment serves no purpose, and other schema variables don't have it.

Probably a leftover.